deploy/docker-compose/README.md

```diff
@@ -0,0 +1,98 @@
+# Docker Compose Deployment
+
+## Quick Start
+
+This is a simple example that shows you how to connect `nm-vllm` with production monitoring via Prometheus and Grafana via Docker Compose.
+
+## Install
+
+Make sure you have Docker and Docker Compose installed.
+- [`docker`](https://docs.docker.com/engine/install/)
+- [`docker compose`](https://docs.docker.com/compose/install/linux/#install-using-the-repository)
```

Comment:
Probably worth keeping this, but I see many indications that `docker compose` is a part of recent `docker` installs.

> With Docker Desktop, Compose V2 is always accessible as docker compose. Additionally, the Use Compose V2 setting is turned on by default, which provides an alias from docker-compose.
> https://docs.docker.com/compose/migrate/

Some indication that maybe only `Docker Desktop` includes it. Either way, food for thought I suppose.